app/routes/comment.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+const router = express.Router()
+const commentController = require('../controllers/comment')
+
+router.post(
+  '/',
+  commentController.comment
+)
+
+router.delete(
+  '/:id',
+  commentController.delete
+)
+
+router.patch(
+  '/:id',
+  commentController.update
+)
+
+router.get(
+  '/:id',
```

Comment:
auth

Reply:
I did not add it because I was testing it. Will do when everything is done.